pkgs/tools/misc/pkgdiff/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, perl, wdiff }:
+stdenv.mkDerivation rec{
+  version = "1.7.2";
+  name = "pkdiff-${version}";
+
+  src = fetchFromGitHub {
+    owner = "lvc";
+    repo = "pkgdiff";
+    rev = version;
+    sha256 = "1ahknyx0s54frbd3gqh070lkv3j1b344jrs6m6p1s1lgwbd70vnb";
+  };
+
+  buildInputs = [ perl ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  makeFlags = [ "prefix=$(out)" ];
```

"more correct", but doesn't really change anything
```suggestion
  makeFlags = [ "prefix=${placeholder "out"}" ];
```